headers/types/common/enums.h

```diff
@@ -3,6 +3,12 @@
 #ifndef HEADERS_TYPES_COMMON_ENUMS_H_
 #define HEADERS_TYPES_COMMON_ENUMS_H_
 
+// Represents one of the two screens the console has
+enum screen {
+    MAIN = 0,
```

Comment:
Prefix enums with a "namespace". Otherwise no other enum can use the name, e.g., `MAIN`. Perhaps `SCREEN_` for `enum screen` and `CANVAS_` for `enum kaomado_canvas_state`.